src/mail-app/app.ts

```diff
@@ -182,7 +182,7 @@ import("./translations/en.js")
 							},
 						}
 					},
-					prepareAttrs: ({ makeViewModel }) => ({ targetPath: "/mail", makeViewModel }),
+					prepareAttrs: ({ makeViewModel }) => ({ targetPath: "/calendar", makeViewModel }),
```

Comment:
this sets the path to calendar inside mail-app/app.ts

Reply:
this was also fixed later